autotst/job/job.py

```diff
@@ -269,15 +271,11 @@ def submit_conformer(self, conformer, restart=False):
 
         self.calculator.conformer = conformer
         ase_calculator = self.calculator.get_conformer_calc()
-        self.write_input(conformer, ase_calculator)
-
         label = conformer.smiles + "_{}".format(conformer.index)
```

Comment:
Is `label` the same as `ase_calculator.label`?  If so, we could use `ase_calculator.label` instead